src/explorer.ts

```diff
@@ -58,6 +64,20 @@ function isResource(obj: KubernetesObject) : obj is KubernetesResource {
     return !!(<KubernetesResource>obj).resourceId;
 }
 
+async function getClusters(kubectl: Kubectl, host: Host) : Promise<KubernetesObject[]> {
+    const shellResult = await kubectl.invokeAsync("config view -o json");
+    if (shellResult.code !== 0) {
+        return [];
+    }
+    const configJson = JSON.parse(shellResult.stdout);
```

Comment:
This is now a JS object, not JSON; consider calling it `config` or `kubectlConfig` or something like that.

Reply:
nice.